SpaghettiCh2/SpaghettiCh2/ViewModels/InstallationViewModel.cs

```diff
@@ -69,6 +77,8 @@ public async Task InstallGame(string exePath)
             {
                 InstallationError?.Invoke(ex, log.ToString());
             }
+
+            await DumpLog(log.ToString());
```

Comment:
Maybe surround with try-catch to prevent failure to log from crashing the whole program?